backend/apid/graphql/asset.go

```diff
@@ -16,11 +16,6 @@ func (*assetImpl) ID(p graphql.ResolveParams) (string, error) {
 	return globalid.AssetTranslator.EncodeToString(p.Source), nil
 }
 
-// Namespace implements response to request for 'namespace' field.
```

Comment:
In part to avoid collisions and for cleanliness, previously the graphql endpoint exposed the organization and environment as its own object.

```graphql
myResource {
  name
  namespace {
    organization
    environment
  }
}
```

Since this is no longer the case the resolver above is rightly removed, the default "alias" resolver satisfies already.